pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -584,8 +590,8 @@ func (asw *actualStateOfWorld) GetAttachState(
 }
 
 // SetVolumeClaimSize sets size of the volume. But this function should not be used from attach_detach controller.
-func (asw *actualStateOfWorld) InitializeClaimSize(volumeName v1.UniqueVolumeName, claimSize *resource.Quantity) {
-	klog.V(5).Infof("no-op InitializeClaimSize call in attach-detach controller.")
+func (asw *actualStateOfWorld) InitializeClaimSize(logger klog.Logger, volumeName v1.UniqueVolumeName, claimSize *resource.Quantity) {
+	logger.Info("no-op InitializeClaimSize call in attach-detach controller")
```

Comment:
This was `V(5)` but is not anymore, why?

Reply:
Sorry, this was a wrong update. has been fixed